extensions/amp-form/0.1/form-validators.js

```diff
@@ -351,6 +361,8 @@ export function getFormValidator(form) {
       return new AsYouGoValidator(form);
     case CustomValidationTypes.ShowAllOnSubmit:
       return new ShowAllOnSubmitValidator(form);
+    case CustomValidationTypes.AsYouGoShowAllOnSubmit:
```

Comment:
some combinations may not make sense like `show-first-on-submit show-all-on-submit` but this would additionally allow `show-first-on-submit as-you-go` which is nice. @cvializ is this possible/easy to implement?